### DISCUSSION TITLE
Global Diversity CFP Day: Asia | Takeaways

### DISCUSSION THREAD

**Discussion by siddharthdayalwal** [EddieHubCommunity/support#950] [Conference Notes]
[Global Diversity CFP Day: Asia](https://ti.to/global-diversity-cfp-day-2021/asia)

:calendar: February 20th, 2021 :round_pushpin: Online

In 2021 you want to get your year off to a flyer and start your journey to becoming a tech conference speaker!

An online event brimming with advice on:

- Putting together your first tech conference proposal
- Figuring out what you should propose a talk about
- Crafting that all important biography
- Structuring your talk
- and a whole lot more!
- All this from seasoned speakers and conference organisers

Start time:
- 10:30 India Standard Time
- 13:00 Singapore Standard Time
- 14:00 Japan Standard Time

End time:
- 16:00 India Standard Time
- 18:30 Singapore Standard Time
- 19:30 Japan Standard Time

**Comment by siddharthdayalwal** [comment]
# **`Welcome to Global Diversity CFP day 2021`** :pray:  

Speaker : **Chen Hui Jing**
> Chen Hui Jing is a self-taught designer and developer with an inordinate love for CSS. Reducing lines of code in her web projects makes her extremely happy. She used to play basketball full-time and launched her web career during downtime between training sessions. Hui Jing is currently a web developer at Shopify.

Gist of the talk :studio_microphone:  

> Aim
- Support  people from underrepresented or marginalised group
- Helping you become a tech conference speaker
- Helping you understand with the process of CFP

> Call For Proposals (CFP)
- Events or conferences looking for speakers start a process called "Call for Proposals". This is an invitation for anyone interested in taking part to submit their idea for a talk or workshop. Proposals are an outline of what you'd like to talk about and helps organisers choose talks for their events.

- Global Diversity CFP day is here to help you craft that proposal and reduce any worry you may have when submitting.

> On Saturday, February 20th 2021 it is the fourth edition of global diversity CFP day - this time online!

There will be 6 streams around the globe encouraging and advising newbie speakers to put together your very first talk proposal and share your own individual perspective on any subject of interest to people in tech.

Each stream will have expert speakers sharing their experience of:
- putting together a talk proposal
- creating that all important biography
- planning and structuring your talk
- delivering your talk
- and a load more tips and advice!

> Thanks to the Sponsors

- [Microsoft](https://www.microsoft.com/)
- [Vito](https://vi.to/)

> [Kindly follow Code of Conduct](https://www.globaldiversitycfpday.com/code-of-conduct)

If you have any questions, reach out to **[@hj_chen](https://twitter.com/hj_chen)**

> Do add your notes as well :smile:

**Comment by siddharthdayalwal** [comment]
# **`My New Year Resolution - Speak at a conference`** :speaking_head:  

Speaker : **Tan Li Hau (He/Him)**
> Frontend Developer at Shopee

Gist of the talk :studio_microphone:  

> Where I started ?

- Attended JS Conf Asia 2019 and as a participant, a thought came up in mind : "If they can speak at a conference, why can't I ?"
- And then I made my New Year Resolution, that I want to speak at a conference I know like JS Conf, React Conf, etc.
- Created excel sheet and collected information about all CFPs wrt my interests.

> Points to more focus on
- Title 
- Abstract
- Speaker notes

---
There were some technical issues later, will add the notes as the recordings are released!

If you have any questions, reach out to **[@lihautan](https://twitter.com/lihautan)**

> Do add your notes as well :smile:

**Comment by siddharthdayalwal** [comment]
# **`Organiser Interviews: 안녕하세요 edition`** :wave: 

Speaker : **Soeun Lee (She/Her)**
> Fullstack Software Engineer at ChartMogul

Gist of the discussion :studio_microphone:  

> Initial conversation 
- Soeun organised [JS Conf Korea](https://jsconfkorea.com/).
- Got motivation from visiting JSConf EU and JSConf Asia.
- Took almost an year to plan for JS Conf Korea.
- 500+ attendance for the JS Conf Korea.

>  Organisers point of view on CFP?
- Topic Idea should be as clear as possible.
- Organisers read every single proposal, got 300-400 CFPs.
- Well explained content in abstract section.

> Pros and Cons for hosting conference online
- Venue rental fee decreased.
- Everyone could attend via internet.
- We were not able to meet face-to-face.
- Tried creating online engagement activities.
- Used Hopin as the platform for conference.

> Tech Community in Korea
- Mostly active in Facebook.
- Number of study groups.
- Contributors translating English to Korean.
- Have various meet-up groups
- Many conferences happenings, JS, DevOps, PyCon, C++, etc.

> Tech Jobs/Work Culture in Korea
- Lots of developer position in Korea.
- Few prefer big tech companies, some prefer working with start-ups.
- Great internet speed.
- Preferred Programming Language in Korea are Java, Kotlin, GOLang, React Library, etc.
- Most companies employees speak Korean but few international companies use English as well during the work.


If you have any questions, reach out to **[@mojosoeun](https://twitter.com/mojosoeun)**

> Do add your notes as well :smile:

**Reply by mojosoeun** [reply]
Hi! @siddharthdayalwal Thank you for creating notes about my interview. Could you change a few things?

> Got motivation from hosting coding contest.

Got motivation from visiting JSConf EU and JSConf Asia

> Great internet speed and thus more taxes

Great internet speed

Thank you!

**Reply by siddharthdayalwal** [reply]
Hey @mojosoeun :wave: 

Thanks for checking that out. I have updated it, let me know if you find any point that needs to be changed. Also do let me know if I missed anything! 

Thanks!

**Comment by siddharthdayalwal** [comment]
# **`Crafting your bio`** :raised_hands:  

Gist of the discussion :studio_microphone:  

> Describing yourself is hard....
- Check the requirements
Usually 100-150 words but this may vary depending on industry.
Doesn't hurt to prepare a short version and a longer version
- What to include ?
Currently _____
Before that _____
Also _____
  1. Professional accomplishments (1-3)
  2. Current occupations and roles
  3. Interests and fun facts
- Examples
  - Hui Jing is a self-taught designer and developer living in Singapore, with an inordinate love for CSS and typography.
  - Jiyeon is a software engineer from Korea working on ML services like face recognition. She is very passionate about exploring new technologies and sharing experience through tech talks. Her favourite language is Korean, but she also likes Python.
- More tips
[Tips for Crafting Your Bio](https://medium.com/women-talk-design/tips-for-crafting-your-bio-5f8fa7b5d234)
[How to become a public speaker in 1 year – Step 5: Craft your bio](https://cattsmall.com/advice/2016/10/03/become-public-speaker-5.html)

---

> Do add your notes as well :smile:

**Comment by siddharthdayalwal** [comment]
# **`Interview sessions: Mabuhay edition`** :wave: 

Speaker : **Kit Valmadrid (She/Her)**
> Project Manager at Globe Telecom

Gist of the discussion :studio_microphone:  

> Initial conversation 
- Philippines is the language of communication.
- Due to pandemic, it has been difficult in Philippines to host events.
- Manages Philippine Web Designers Organization [@PWDO](https://twitter.com/pwdo)

> Thoughts on organising conference with consistency
- Get enough experience as it doesn't happens at once.
- Thinking on various themes for every year.
- Considering speakers and working around it essential.
- Post Event work is must, sending slides, survey, feed-backs, etc.
- Video recordings for the speaker sessions.
- Pre Event efforts are much much more.

> Views on Speaker Selection
- Team have their own speaker wish-list.
- A nomination form for speakers.
- Also a form which one can fill to show the interest.
- Focuses on foreign speakers to have diverse group of people.
- Cost also needs to be considered.
- Announces CFPs.

> Support to New Speakers 
- Start via inviting them to meet-ups/mini-events.
- Have Philippines Tech Community.
- Inclusive and Welcoming environment.
- Motivates and help new people to come forward and share knowledge.
- This works as a training phase for new speakers.

> PWD Organising Team
- They do have a good mix of genders in their.
- Most of the leaders in the organization are woman.
- FFC happened mostly online and less meet-ups.
- Organisers from different region shares their idea.

> Advice to first time speakers
- Start from school, as students are having less expectation from you.
- Then you can proceed to more professional.
- Keep trying and practicing.
- Be with like-minded people.
- Choose a set of community having same mind-set.
- Join your local tech communities.

> Tech Jobs/Work Culture in Philippine
- Low pay scale here.
- Different pay scale for different regions based on chart which is followed.
- Mostly developers and designers are working for Agencies.
- Agencies would be preferred than any other companies.

If you have any questions, reach out to **[@sinongkit](https://twitter.com/sinongkit)**

> Do add your notes as well :smile:

**Comment by siddharthdayalwal** [comment]
# **`How I ended up on the JSConf Asia stage`** :rocket:   

Speaker : **Thai Pangsakulyanont (He/Him)**
> Building an open-source web-based rhythm game [@bemusegame](http://twitter.com/bemusegame)

Gist of the talk :studio_microphone:  

> Experience
- Starting from 2015
  - I was an introvert in the beginning.
  - Unconference helped me get better in speaking at events.
  - If you're new, go and attend Unconferences & Open Spaces, just be yoruself.
- Next was 2016
  - I started volunteering.
  - Organised first React Bangkok.
  - Nat recommended JS Conf Asia.
  - Proposal got rejected.
  - Did networking, made friends, learned a lot.
- 2017 : JS Conf Asia announced 
  - Proposal accepted but with different topic.
- 2018 : JS Conf Asia 2018.
  - Submitted CFP at JS Conf Asia 2019
  - My Production Process
    - I write from start to finish (without outline)
    - I do efficient work more efficiently
    - I practice multiple times.
    - Checking with the duration and cutting it to a period.
    - Enjoying the conference and taking breaks.
    - And finally I did it.
    - Don't forget to click enough photos.
- 2019
  - Proposal accepted at JS Conf Asia 2019

> Giving a talk is best conversation starter.

> If you're new to speaking
- Networking is important
- Alteast one new person each event
- Join a random circle
- Find your conversation started.
- Volunteer in organising events.
- Take a chance sometimes.
- Find your presentation style.
- Experiment and explore.

> Having a story line in a talk helps, it makes it interesting and entertaining.

If you have any questions, reach out to **[@dtinth](https://twitter.com/dtinth)**

> Do add your notes as well :smile:

**Comment by siddharthdayalwal** [comment]
# **`Interview sessions: สวัสดีค่ะ edition`** :wave: 

Speaker : **Thai Pangsakulyanont (He/Him)**
> Building an open-source web-based rhythm game [@bemusegame](http://twitter.com/bemusegame)

Gist of the discussion :studio_microphone:  

> Initial conversation 
- Speaker at JS Conf Asia 2018 and JS Conf Asia 2019.
- Organised React Bangkok Event.
- Translated Thai to English in the conference.

> Speaker Selection Process
- Proposals that can be great for the conference.
- Informative and detailed.
- Anonymous CFP.
- Tell as much as you.

> Community Culture in Bangkok
- We do have Developer Communities here.
- Shinmai is great for events, meet-ups, etc.

If you have any questions, reach out to **[@dtinth](https://twitter.com/dtinth)**

> Do add your notes as well :smile:

**Comment by siddharthdayalwal** [comment]
# **`Interview sessions: 大家好 edition`** :wave: 

Speaker : **Annie Huang (She/Her)**
> Community Advocate, APAC at GitLab 

Gist of the discussion :studio_microphone:  

> Initial conversation 
- Co-Organiser of GDG Taiwan.
- Women TechMakers Ambassador Taiwan.
- Running a lot of developer community events.

> Speaker Preference
- Speaker with experience.
- Audience focused topic with practical approach.

- Rehearsals helps overcoming stage/audience fear and makes speaker more comfortable.
- Ask for help, not to feel shy in asking.

> Tech Communities in Taiwan
- Taipei having great number of events happening.
- Very welcoming environment.
- Developers are passionate about telling you all the stories.
- Taitung GDG is emerging community and have great culture.

> Tech Jobs/Work Culture in Taiwan
- Few companies are open to Remote Work.
- But not all of them prefer Remote.

If you have any questions, reach out to **[@annie_huang1018](https://twitter.com/annie_huang1018)**

> Do add your notes as well :smile:

**Comment by siddharthdayalwal** [comment]
# **`Organising your talk`** :rocket:   

Speaker : **Jiyeon Seo (She/Her)**
> Software Engineer

Gist of the talk :studio_microphone:  

> Imagine you are just accepted for the talk!!

- Yay, you are selected
- About the topic, I know but I might not know
- I don't know this and that neither.
- Should I give up?
- What now ?

> How do I start ?

1. **Contents**
i. Don't make a whole story at once
    - Ideas first : Don't care about story, steps, unclear or anything.
    - Structure them
    - Ideas again
      - Repeat above steps, till story --> slide
    - Slides
      - Head : Title + Motivation + Introduction
      - Body : What I want to achieve ? + Agenda + Content (keywords)
      - Footer : Conclusion + End Page

    ii. SOLID
![image](https://user-images.githubusercontent.com/41017419/108590945-26e3a180-738c-11eb-955c-ae338e54e2a9.png)
![image](https://user-images.githubusercontent.com/41017419/108591002-732ee180-738c-11eb-8d2c-bc2967b7212f.png)

    iii. Code Snippet : Highlight & Simple
    - Well structured
    - Few lines
    - Properly highlighted
    - Adding Keywords
    - Adding GitHub Link
![image](https://user-images.githubusercontent.com/41017419/108591027-95c0fa80-738c-11eb-8e8c-231637e48fb9.png)

    iv. Practice
![image](https://user-images.githubusercontent.com/41017419/108591089-d15bc480-738c-11eb-8227-0e6d3e16a832.png)

After all this : 
![image](https://user-images.githubusercontent.com/41017419/108591109-f0f2ed00-738c-11eb-84ff-dfcf0b272180.png)

2. **On Stage**
    i. Script ? No Script ?
      - Reading Script may cause
        - Poor interaction with audience
        - Loss of their concentration

    ii. Make a hint for next slide

    iii. Consideration for online talks
      - Network, Network, Network!!!

> Commit first and figure it out later.
- There will be many ups and downs in your confidence the entire period
![image](https://user-images.githubusercontent.com/41017419/108591391-8f338280-738e-11eb-9ed0-10801013f455.png)

> Feel like failed??
- **It's because you knows how to do it better next time.**

- - -
If you have any questions, reach out to **[@seojeee](https://twitter.com/seojeee)**

> Do add your notes as well :smile:

**Comment by siddharthdayalwal** [comment]
# **`Prerecording Talks for Online Conferences`** :microphone:    

Speaker : **Shawn Wang (He/Him)**
> Working on Developer Experience for Netlify, Amazon Web Services, and Temporal

Gist of the talk :studio_microphone:  

> 15 Tips to Prerecord your conference talk.

- Record Yourself 
    i. Script
    ii. Lighting
    iii. Background
    iv. Camera
    v. Microphone

- Record Screen
    vi. Bigger
    vii. Clearer
    viii. Keyboard
    ix. Face
    x. Slides

- Editing your Video
    xi. Layouts
    xii. Transitions
    xiii. Speed-Up
    xiv. Super-impose
    xv. Breadcrumbs

> [CFP Advice by Shawn](https://www.swyx.io/cfp-advice/)

If you have any questions, reach out to **[@swyx](https://twitter.com/swyx)**

> Do add your notes as well :smile:

**Comment by siddharthdayalwal** [comment]
# ✍️ Call for proposals ✍️

- [Web Directions (conference)](https://webdirections.org/events/)
- [CFP Land](https://www.cfpland.com/)
- [Confs.tech](https://confs.tech/)
- [Calling All Papers (Twitter)](https://twitter.com/callingallpaper)
- [CallbackWomen (Twitter)](https://twitter.com/CallbackWomen)
- [PaperCall event directory](https://www.papercall.io/events)

# 📋 Useful links 📋

- [Don’t feel like an expert? Share anyway.](https://modus.medium.com/dont-feel-like-an-expert-share-anyway-661f2f8cd038)
- [How To Rock The Call For Papers And Get Your Talk Picked](https://www.jonobacon.com/2019/01/21/get-your-cfp-session-picked/)
- [How to Manage Your Fear of Public Speaking](https://medium.com/better-humans/how-to-manage-your-fear-of-public-speaking-92964d531147)
- [Advent speaker tips](https://be.noti.st/blog/category/advent-speaker-tips), [Speaking Tips](https://be.noti.st/blog/category/speaking-tips)
- [Global Diversity: helping designers break into the speaker circuit](https://uxdesign.cc/global-diversity-helping-designers-break-into-the-speaker-circuit-7cd4c240cc6c)
- [The Ultimate Guide To Memorable Tech Talks](https://nnja.medium.com/the-ultimate-guide-to-memorable-tech-talks-e7c350778d4b)
- [Becoming a Tech Speaker](https://css-irl.info/becoming-a-tech-speaker/)
- [All about giving technical talks on stage](https://devrelweekly.com/issues/55)

**Comment by eddiejaoude** [comment]
This is so EPIC ❤️ 💥

**Reply by siddharthdayalwal** [reply]
Thanks @eddiejaoude :heart: :relieved:
